rollup.config.mjs

```diff
@@ -1,14 +1,17 @@
 import typescript from '@rollup/plugin-typescript';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
+import shebang from 'rollup-plugin-shebang-bin'
 
+/** @type {import('rollup').RollupOptions} */
 const baseConfig = {
   output: {
     dir: 'dist',
     format: 'esm'
   },
   plugins: [
     nodeResolve(),
-    typescript({ tsconfig: './tsconfig.json' })
+    typescript({ tsconfig: './tsconfig.json' }),
+    shebang({ include: "src/clientApp.ts" }),
```

Comment:
I wonder if this can be done with a banner in the output section instead of adding the sheband dependency
```
banner: '#!/usr/bin/env node'
```

Reply:
It's not only about the banner, we need a chmod +x to happen to the output.